docs/guidelines-for-setting-up-a-local-workspace.md

```diff
@@ -45,55 +45,159 @@ Under Ubuntu Linux, you can follow the [documentation from the Ubuntu Community]
 * Go to the newly created jabref folder: `cd jabref`
 * Generate additional source code: `./gradlew assemble`
 * Start JabRef: `./gradlew run`
+  * The JabRef's GUI should now start up.
 
 ## Configure your IDE
 
-### IntelliJ
+### Setup for IntelliJ
 
 1. Open `jabref/build.gradle` as a project
 2. Enable annotation processors:
-   * File -> Settings -> Compiler -> Annotation processors -> Check "Enable annotation processing"
-3. Configure module settings: Right click on project -> Open Module Settings
-   * Ensure that the projects SDK is Java 13: Project Settings -> Project -> Project SDK: Choose Java 13
-   * Ensure that standard SDK is Java 13: Platform Settings -> SDK -> Choose Java 13
-4. Specify additional compiler arguments: File -> Settings -> Build, Execution, Deployment -> Compiler -> Java Compiler -> Under "Override compiler parameters per-module" add the following compiler arguments for the `JabRef.main` module:
-   ```text
-   --patch-module test=fastparse_2.12-1.0.0.jar
-   --patch-module test2=fastparse-utils_2.12-1.0.0.jar
-   --patch-module test3=sourcecode_2.12-0.1.4.jar
-   --add-exports javafx.controls/com.sun.javafx.scene.control=org.jabref
-   --add-exports org.controlsfx.controls/impl.org.controlsfx.skin=org.jabref
-   --add-exports javafx.graphics/com.sun.javafx.scene=org.controlsfx.controls
-   --add-exports javafx.graphics/com.sun.javafx.scene.traversal=org.controlsfx.controls
-   --add-exports javafx.graphics/com.sun.javafx.css=org.controlsfx.controls
-   --add-exports javafx.controls/com.sun.javafx.scene.control.behavior=org.controlsfx.controls
-   --add-exports javafx.controls/com.sun.javafx.scene.control=org.controlsfx.controls
-   --add-exports javafx.controls/com.sun.javafx.scene.control.inputmap=org.controlsfx.controls
-   --add-exports javafx.base/com.sun.javafx.event=org.controlsfx.controls
-   --add-exports javafx.base/com.sun.javafx.collections=org.controlsfx.controls
-   --add-exports javafx.base/com.sun.javafx.runtime=org.controlsfx.controls
-   --add-exports javafx.web/com.sun.webkit=org.controlsfx.controls
-   --add-exports javafx.graphics/com.sun.javafx.css=org.controlsfx.controls
-   --add-exports javafx.controls/com.sun.javafx.scene.control.behavior=com.jfoenix
-   --patch-module org.jabref=build/resources/main
+   * File -> Settings -> Compiler -> Annotation processors:
```

Comment:
This was already missing before the other changes, but it would be nice, if you could fix it here, since it is printed there too:
... Settings -> Build, Execution, Deployment -> Compiler ...